assets/js/group-details.js

```diff
@@ -0,0 +1,467 @@
+/* global jQuery:false, wpApiSettings:false */
+
+jQuery.ajaxSetup({
+  beforeSend: function(xhr) {
+    xhr.setRequestHeader('X-WP-Nonce', wpApiSettings.nonce);
+  },
+})
+
+jQuery(document).ready(function($) {
+function save_field_api(groupId, post_data){
```

Comment:
The lack of indentation here makes me nervous.

Reply:
@burntsun Kinda like falling off a cliff?

Reply:
Heh.